src/WordDaze.Client/Features/Home/Home.razor.cs

```diff
@@ -7,20 +7,20 @@
 using WordDaze.Shared;
 using System.Linq;
 
-namespace WordDaze.Client.Features.Home
+namespace WordDaze.Client.Features
```

Comment:
Just a nit pick thing but could we keep the name spacing scoping objects to feature areas. I know this leads to more using statements in the _Imports.razor but I think it keeps things cleaner.

Reply:
Absolutely. This is a leftover from getting it to work on the new version. Was moving and renaming stuff